src/core_plugins/kibana/public/management/sections/objects/_view.js

```diff
@@ -199,11 +200,17 @@ uiModules.get('apps/management')
           }
           const confirmModalOptions = {
             onConfirm: doDelete,
-            confirmButtonText: 'Delete',
-            title: 'Delete saved Kibana object?'
+            confirmButtonText: i18n.translate('kbn.management.objects.confirmModalOptions.deleteButtonLabel', {
```

Comment:
We can use `I18n` AngularJS service instead of core service. Please inject it and use in this file.